sdk/storage/storage-blob/src/generatedModels.ts

```diff
@@ -92,6 +92,7 @@ export {
   EncryptionAlgorithmType,
   GeoReplication,
   GeoReplicationStatusType,
+  KnownEncryptionAlgorithmType,
```

Comment:
What's this property for?

Reply:
It's used for describe `EncryptionAlgorithmType`'s known value.